### ISSUE TITLE
Feature request: filter tables in browser

### ISSUE THREAD

**Issue by osluys** [fabianfiorotto/quick-query#41]
Thanks for creating this package: it is exactly what I was looking for. I was using [DBeaver](http://dbeaver.jkiss.org/) before finding your package and one feature I miss is the ability to filter tables in the browser. I have upwards of 100+ tables, but I only need to see a handful of them.

Ideally, I would be able to do the following:
* Quickly turn on/off filtering globally
* Add multiple filters with wildcard support
* Automatically sort filters when editing filters (missing from DBeaver)
* Create filter groups (missing from DBeaver)
* Quickly turn on/off individual group/filter (missing from DBeaver)

Maybe a lot to ask for, but I would also be happy with just basic filtering. Thanks again.

**Comment by fabianfiorotto** [comment]
You already have a fuzzy finder to do a quick select and it scrolls to the table in the browser if it's open. In the command palette type "Find table to select" or press <kbd>ctrl</kbd>+<kbd>alt</kbd>+<kbd>T</kbd> if you haven't overridden the keymap. I don't know if that is what you are looking for.

**Comment by osluys** [comment]
Not exactly. I was thinking more of limiting the returned list of tables in the browser to only the ones that match a pattern. I know about fuzzy finder, but I prefer to actually see a list of tables that I need to work with, as I find that less taxing on my brain than trying to remember the names of tables. The code I work with uses somewhat obscure table names, and I find myself frequently having to look up the table name.

**Comment by fabianfiorotto** [comment]
This package offers the table names as suggestions. If you only need the table names, just type the name in an editor and a list of tables will appear. It only works when the current syntax is SQL.
